### PR TITLE
feat: Don't include or_dev when building in kubernetes

### PR DESCRIPTION
**Pull request by plisy**
## What this PR does / why we need it

When running in devspace, build should not be set up for dev/devenv as that's not available.

## Jira ID

[XX-XX]

### CODE REVIEW

File: root/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ LDFLAGS             ?= -w -s -X github.com/getoutreach/gobox/pkg/app.Version=$(A
 GOFLAGS             :=
 GOPRIVATE           := github.com/$(ORG)/*
 GOPROXY             := https://proxy.golang.org
-GO_EXTRA_FLAGS      := -v -tags=or_dev
+GO_EXTRA_FLAGS      := -v $(shell [[ ! -z "$KUBERNETES_SERVICE_HOST" ]] && printf "-tags=or_dev")
```
Comment (jaredallard) [inline root/Makefile:17]: nit `! -z` -> `-n`

Comment (jaredallard) [reply root/Makefile:17]:
```suggestion
GO_EXTRA_FLAGS      := -v $(shell [[ -n "$KUBERNETES_SERVICE_HOST" ]] && printf "-tags=or_dev")
```

Comment (jaredallard) [reply root/Makefile:17]: Wait, we want `-z` here, right? If `KUBERNETES...` is empty, use `or_dev`. Not the other way around?

Comment (jaredallard) [reply root/Makefile:17]: or am I misreading the one-liner, I hate bash one-line if statements

Comment (plisy) [reply root/Makefile:17]: this seems to do what we want, if `KUBERNETES...` is empty, add the `or_dev`. I hate bash.

Comment (jaredallard) [reply root/Makefile:17]: Hmmmm, ok new suggestion (fun fact don't need to quote in if statement)

ref: https://stackoverflow.com/questions/18096670/what-does-z-mean-in-bash

```suggestion 
GO_EXTRA_FLAGS      := -v $(shell if [[ -z $KUBERNETES_SERVICE_HOST ]]; then printf "-tags=or_dev"; fi)
```

**Pull request merged**